src/newcharacter.cpp

```diff
@@ -2828,12 +2827,12 @@ tab_direction set_description( avatar &you, const bool allow_reroll,
                     .only_digits( true );
                     const int result = popup.query_int();
                     if( result != 0 ) {
-                        you.set_base_height( clamp( popup.query_int(), 145, 200 ) );
+                        you.set_base_height( clamp( result, 145, 200 ) );
                     }
                     break;
                 }
                 case char_creation::BLOOD: {
-                    popup.title( _( "Enter blood type (omit Rh):" ) )
+                    popup.title( _( "Enter blood type (O, A, B or AB):" ) )
```

Comment:
I think `ui_list` is better suited for this kind of selection from a fixed list.